src/qt/android/.gitignore

```diff
@@ -3,7 +3,5 @@
 /gradle/wrapper
 /gradlew*
 /libs
-/res/layout
-/res/values*
```

Comment:
While here, maybe s|`/gradle/wrapper`|`/gradle`| as `/gradle/wrapper` is the only subdirectory in `/gradle`, no?